pages/common/phpcbf.md

```diff
@@ -15,9 +15,9 @@
 
 `phpcbf {{path/to/directory}} --standard {{standard}}`
 
-- Specify file extension(s) to include when sniffing:
+- Specify comma-separated file extensions to include when sniffing:
 
-`phpcbf {{path/to/directory}} --extensions {{file_extension(s)}}`
+`phpcbf {{path/to/directory}} --extensions {{file_extension}},{{other_extension}}`
```

Comment:
I think it's fine to keep the token as `{{file_extension(a)}}` if you have just updated the descriptions.